src/ZendViewRendererFactory.php

```diff
@@ -62,8 +62,10 @@ public function __invoke(ContainerInterface $container)
             100
         );
 
-        // Create the renderer
-        $renderer = new PhpRenderer();
+        // Create, or Retrieve the renderer from the container
+        $renderer = ($container->has(PhpRenderer::class))
+                  ? $container->get(PhpRenderer::class)
```

Comment:
Change to RendererInterface::class.

Reply:
Initially, I thought that using `RendererInterface` was absolutely the right thing, but the problem is that a `PhpRenderer` is assumed elsewhere, such as [`ZendViewRenderer::__construct()`](https://github.com/zendframework/zend-expressive-zendviewrenderer/blob/bd5acd27e3409f213531355188a8a0cfca2526e5/src/ZendViewRenderer.php#L76), there's no `resolver()` method in `RendererInterface`, so should the constructor signature for `ZendViewRenderer` be changed to consume a `PhpRenderer` or should we leave this pull so that it type hints against the concrete class?

Reply:
In this case, concrete class, as there are even methods we're using that are specific to the `PhpRenderer` implementation. I think `PhpRenderer` is appropriate here.